ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -5734,3 +5734,40 @@ def test_torch_special_and(
         frontend=frontend,
         on_device=on_device,
     )
+
+    # bitwise_xor
+
```

Comment:
Not necessarily a functional thing but it would be nice to have the indentation of the comment aligned with the rest of the code. 🙂

Reply:
Sorry for this, I thought it will get handled by the pre-commit by itself, anyways I have done the changes

Reply:
No worries! Thanks!